code/src/components/Album.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import './Album.css'
+
+export const Album = (props) => { 
```

Comment:
This component could be divided to many smaller components. If there are more than 20 JSX lines of code then it's a clear information you could probably do it better